interface-definitions/service_monitoring_telegraf.xml.in

```diff
@@ -0,0 +1,109 @@
+<?xml version="1.0"?>
+<interfaceDefinition>
+  <node name="service">
+    <children>
+      <node name="monitoring">
+        <properties>
+          <help>Monitoring services</help>
+          <priority>1280</priority>
+        </properties>
+        <children>
+          <node name="telegraf" owner="${vyos_conf_scripts_dir}/service_monitoring_telegraf.py">
+            <properties>
+              <help>Service telegraf</help>
+            </properties>
+            <children>
+              <node name="authentication">
+                <properties>
+                  <help>Authentication parameters</help>
+                </properties>
+                <children>
+                  <leafNode name="organization">
+                    <properties>
+                      <help>Authentication organization for InfluxDB v2 [REQUIRED]</help>
+                    </properties>
+                  </leafNode>
+                  <leafNode name="token">
+                    <properties>
+                      <help>Authentication token for InfluxDB v2 [REQUIRED]</help>
+                      <valueHelp>
+                        <format>txt</format>
+                        <description>Authentication token</description>
+                      </valueHelp>
+                      <constraint>
+                        <regex>^[a-zA-Z0-9-_]{86}==$</regex>
+                      </constraint>
+                      <constraintErrorMessage>Token contains illegal characters or\nless then 8 characters or\nexceeds 100 character limitation.</constraintErrorMessage>
```

Comment:
Maybe reword to something like "Token must be 8-100 characters long and must contain only [a-zA-Z0-9-_] characters" to make the error more "actionable"?